Can't install gcc48 homebrew osx 10.10

When running `rvm requirements`, i get the following error:

```bash
rvm requirements
Checking requirements for osx.
Installing requirements for osx.
Updating system.....
Installing required packages: gcc48.....
Error running 'requirements_osx_brew_libs_install gcc48',
showing last 15 lines of /Users/martintreurnicht/.rvm/log/1426665110/package_install_gcc48.log
+ case "$1" in
+ [[ -t 1 ]]
+ return 1
+ printf %b 'There were package installation errors, make sure to read the log.

Try `brew tap --repair` and make sure `brew doctor` looks reasonable.

Check Homebrew requirements https://github.com/Homebrew/homebrew/wiki/Installation\n'
There were package installation errors, make sure to read the log.

Try `brew tap --repair` and make sure `brew doctor` looks reasonable.

Check Homebrew requirements https://github.com/Homebrew/homebrew/wiki/Installation
+ case "$_system_version" in
+ return 1
Requirements installation failed with status: 1.

```

installing gcc48 via homebrew fails too, but gcc49 works. Possible to upgrade to gcc49 or resolve this issue?

Here is the full log:

``` bash
[2015-03-18 09:52:47] requirements_osx_brew_libs_install
requirements_osx_brew_libs_install () 
{ 
    brew unlink "$@" || true;
    brew install "$@" || { 
        \typeset ret=$?;
        rvm_warn "There were package installation errors, make sure to read the log.

Try \`brew tap --repair\` and make sure \`brew doctor\` looks reasonable.

Check Homebrew requirements https://github.com/Homebrew/homebrew/wiki/Installation";
        case "$_system_version" in 
            10.6)
                rvm_warn "
On OSX 10.6 instead of command line tools install:
    https://github.com/downloads/kennethreitz/osx-gcc-installer/GCC-10.6.pkg"
            ;;
        esac;
        return $ret
    }
}
current path: /Users/martintreurnicht
GEM_HOME=/Users/martintreurnicht/.rvm/gems/ruby-2.2.1
PATH=/Users/martintreurnicht/.rvm/gems/ruby-2.2.1/bin:/Users/martintreurnicht/.rvm/gems/ruby-2.2.1@global/bin:/Users/martintreurnicht/.rvm/rubies/ruby-2.2.1/bin:/Users/martintreurnicht/.gvm/vertx/current/bin:/Users/martintreurnicht/.gvm/springboot/current/bin:/Users/martintreurnicht/.gvm/lazybones/current/bin:/Users/martintreurnicht/.gvm/jbake/current/bin:/Users/martintreurnicht/.gvm/groovyserv/current/bin:/Users/martintreurnicht/.gvm/groovy/current/bin:/Users/martintreurnicht/.gvm/griffon/current/bin:/Users/martintreurnicht/.gvm/grails/current/bin:/Users/martintreurnicht/.gvm/gradle/current/bin:/Users/martintreurnicht/.gvm/glide/current/bin:/Users/martintreurnicht/.gvm/gaiden/current/bin:/Users/martintreurnicht/.gvm/crash/current/bin:/Users/martintreurnicht/.gvm/asciidoctorj/current/bin:/usr/local/sbin:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin::/opt/X11/bin:/Users/martintreurnicht/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/martintreurnicht/.rvm/bin
GEM_PATH=/Users/martintreurnicht/.rvm/gems/ruby-2.2.1:/Users/martintreurnicht/.rvm/gems/ruby-2.2.1@global
command(2): requirements_osx_brew_libs_install gcc48
+ brew unlink gcc48
Error: No such keg: /usr/local/Cellar/gcc48
+ true
+ brew install gcc48
==> Installing gcc48 from homebrew/homebrew-versions
==> Downloading http://ftpmirror.gnu.org/gcc/gcc-4.8.3/gcc-4.8.3.tar.bz2
Already downloaded: /Library/Caches/Homebrew/gcc48-4.8.3.tar.bz2
==> ../configure --build=x86_64-apple-darwin14.1.0 --prefix=/usr/local/Cellar/gcc48/4.8.3 --enable-languages=c,c++,objc,obj-c++ --program-suffix=-4.8 --with-gmp=/usr/local/opt/gmp4 --with-mpfr=/usr/local/opt/mpfr2 --with-mpc=/usr/local/opt/libmpc08 --with-cloog=/usr/local/opt/cloog018 --with-isl=/usr/local/opt/isl011 --with-system-zlib --enable-version-specific-runtime-libs --enable-libstdcxx-time=yes --enable-stage1-checking --enable-checking=release --enable-lto --disable-werror --with-pkgversion=Homebrew gcc48 4.8.3 --with-bugurl=https://github.com/Homebrew/homebrew-versions/issues --enable-plugin --disable-nls --enable-multilib
==> make bootstrap
make[4]: *** [all-multi] Error 2
make[3]: *** [all-recursive] Error 1
make[2]: *** [all-stage1-target-libsanitizer] Error 2
make[1]: *** [stage1-bubble] Error 2
make: *** [bootstrap] Error 2

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/homebrew/homebrew-versions/issues

+ typeset ret=1
+ rvm_warn 'There were package installation errors, make sure to read the log.

Try `brew tap --repair` and make sure `brew doctor` looks reasonable.

Check Homebrew requirements https://github.com/Homebrew/homebrew/wiki/Installation'
+ rvm_pretty_print stdout
+ case "${rvm_pretty_print_flag:=auto}" in
+ case "${TERM:-dumb}" in
+ case "$1" in
+ [[ -t 1 ]]
+ return 1
+ printf %b 'There were package installation errors, make sure to read the log.

Try `brew tap --repair` and make sure `brew doctor` looks reasonable.

Check Homebrew requirements https://github.com/Homebrew/homebrew/wiki/Installation\n'
There were package installation errors, make sure to read the log.

Try `brew tap --repair` and make sure `brew doctor` looks reasonable.

Check Homebrew requirements https://github.com/Homebrew/homebrew/wiki/Installation
+ case "$_system_version" in
+ return 1

```

--- a/scripts/functions/requirements/osx_brew
+++ b/scripts/functions/requirements/osx_brew
@@ -197,12 +197,13 @@ run 'brew doctor' and make sure latest '${openssl_name}' is installed properly."
 requirements_osx_brew_libs_default_check_gcc()
 {
   if
-    requirements_detect_installed gcc46 gcc47 gcc48 apple-gcc42
+    requirements_detect_installed gcc46 gcc47 gcc48 gcc49 apple-gcc42
   then
     case "${packages_installed[*]}" in
       (*gcc46*) export CC="$(brew --prefix gcc46      )/bin/gcc-4.6" ;;
       (*gcc47*) export CC="$(brew --prefix gcc47      )/bin/gcc-4.7" ;;
       (*gcc48*) export CC="$(brew --prefix gcc48      )/bin/gcc-4.8" ;;
+      (*gcc49*) export CC="$(brew --prefix gcc49      )/bin/gcc-4.9" ;;
       (*gcc42*) export CC="$(brew --prefix apple-gcc42)/bin/gcc-4.2" ;;
       (*)       return 1 ;;
     esac
@@ -214,7 +215,7 @@ requirements_osx_brew_libs_default_check_gcc()
 requirements_osx_brew_libs_default_add_gcc_v_auto()
 {
   if   __rvm_version_compare "${_system_version}" -ge 10.10
-  then requirements_osx_brew_libs_default_add_gcc_v 4.8
+  then requirements_osx_brew_libs_default_add_gcc_v 4.9
   elif __rvm_version_compare "${_system_version}" -ge 10.7
   then requirements_osx_brew_libs_default_add_gcc_v 4.6
   else requirements_osx_brew_libs_default_add_gcc_v 4.2
@@ -228,6 +229,7 @@ requirements_osx_brew_libs_default_add_gcc_v()
     (4.6) requirements_osx_brew_libs_default_add_gcc "$2" gcc-4.6 gcc46       homebrew/versions ;;
     (4.7) requirements_osx_brew_libs_default_add_gcc "$2" gcc-4.7 gcc47       homebrew/versions ;;
     (4.8) requirements_osx_brew_libs_default_add_gcc "$2" gcc-4.8 gcc48       homebrew/versions ;;
+    (4.9) requirements_osx_brew_libs_default_add_gcc "$2" gcc-4.9 gcc49       homebrew/versions ;;
     (*)
       rvm_error "Do not know how to check/install gcc '$1'."
       return 1
@@ -284,7 +286,7 @@ requirements_osx_brew_libs_default()
           ;;
       esac
       ;;
-    (*gcc-4.2|*gcc-4.6|*gcc-4.7|*gcc-4.8)        # version                  full_path
+    (*gcc-4.2|*gcc-4.6|*gcc-4.7|*gcc-4.8|*gcc-4.9) # version                   full_path
       requirements_osx_brew_libs_default_add_gcc_v ${selected_compiler##*-} "${selected_compiler}" || return $?
       ;;
     (*)

--- a/scripts/functions/requirements/osx_fink
+++ b/scripts/functions/requirements/osx_fink
@@ -87,7 +87,7 @@ requirements_osx_fink_gcc_version_detect()
   elif 
     __rvm_version_compare "${_system_version}" -le 10.10
   then 
-    _libs+=( gcc48 )
+    _libs+=( gcc49 )
   else # 10.10
     fink_libs+=( gcc49 )
   fi

--- a/scripts/functions/requirements/osx_port
+++ b/scripts/functions/requirements/osx_port
@@ -71,7 +71,7 @@ requirements_osx_port_libs_default()
         __rvm_compiler_is_llvm # install gcc only if not yet available
       then
         if __rvm_version_compare "${_system_version}" -ge 10.8
-        then port_libs+=( gcc48 )
+        then port_libs+=( gcc49 )
         else port_libs+=( apple-gcc42 )
         fi
       fi
@@ -157,6 +157,9 @@ requirements_osx_port_after()
       ;;
     (*gcc48*)
       export CC=gcc-mp-4.8
+      ;;
+    (*gcc49*)
+      export CC=gcc-mp-4.9
       ;;
   esac
 

